rcgen/src/sign_algo.rs

```diff
@@ -5,14 +5,14 @@ use yasna::DERWriter;
 use yasna::Tag;
 
 #[cfg(feature = "crypto")]
-use crate::ring_like::signature::{self, EcdsaSigningAlgorithm, EdDSAParameters};
+use crate::ring_like::signature::{self, EcdsaSigningAlgorithm, EdDSAParameters, RsaEncoding};
```

Comment:
I was confused by the lack of a `cfg` guard for this import, but I guess ring does actually have this type (it just doesn't allow signing with it).

Reply:
Yeah ring has this type too and it in fact requires it for signing, so we already use it where we do signing, but now the PR adds it to `SignatureAlgorithm`'s `SignAlgo::Rsa`. There is some potential followup where we do the signing to use the contained `RsaEncoding` directly instead of manually reconstructing it via an if-chain.